CHANGES.md

```diff
@@ -1,11 +1,22 @@
 # Changes
 
-### v0.2.0, 06-20-2018
+### v0.3.0, 2019-11-01
+
+* Major changes to the API. Please refer to the [ReadMe](readme.md) file for information on how to upgrade
+
+* Added new algorithms
+  * Threshold Optimization
+  * Grid Search
```

Comment:
do you want to say that expgrad changed significantly in the way we call it?

Reply:
I've called that out on the line above